src/index.js

```diff
@@ -399,6 +403,35 @@ class AvatarEditor extends React.Component {
     this.props.onMouseMove()
   }
 
+  rotate(angle) {
+    // Normalize angle (only 90/180/270 is allowed)
+    angle %= 360;
+    angle = (angle < 0) ? angle + 360 : angle;
+    angle -= angle % 90;
+    if (!angle) {
+        return;
+    }
+    const canvas = document.createElement('canvas');
+    const context = canvas.getContext('2d');
+    const imageState = this.state.image;
+    const iWidth = imageState.resource.width;
+    const iHeight = imageState.resource.height
+    canvas.width = iWidth;
+    canvas.height = iHeight;
+    // if 90 or 270 - switch width and height
+    if ((angle % 180) !== 0) {
+        canvas.width = iHeight;
+        canvas.height = iWidth;
+    }
+    context.save();
+    context.translate((canvas.width / 2), (canvas.height / 2));
+    context.rotate((angle * Math.PI / 180));
+    context.translate(-(iWidth / 2), -(iHeight / 2));
+    context.drawImage(imageState.resource, 0, 0);
+    context.restore();
+    this.handleImageReady(canvas);
```

Comment:
This call will generate a spurious call to `this.props.onLoadSuccess` whenever the image is rotated.

(haven't tested yet: I may be wrong)